scripts/genie.lua

```diff
@@ -137,6 +142,8 @@ MODULE_DIR = path.getabsolute("../")
 BGFX_DIR   = path.getabsolute("..")
 BX_DIR     = os.getenv("BX_DIR")
 BIMG_DIR   = os.getenv("BIMG_DIR")
+DAWN_DIR   = os.getenv("DAWN_DIR")
+print(DAWN_DIR)
```

Comment:
nit: stray print?

Reply:
Also this block should be moved into `if _OPTIONS["webgpu"] then` block.

If DAWN has some environment variable, then originals should be used.

Reply:
Dawn doesn't have environment variables (apart from the Vulkan loader ones when using that backend).

Reply:
Yeah, this is meant to be set by the user. I don't have a smarter and cheaper way to bridge bgfx GENie based build with Dawn's CMake one than letting the user set this env var

Reply:
Addressed both your comments